src/openlifu/xdc/transducer.py

```diff
@@ -153,8 +153,6 @@ def rescale(self, units):
         if self.units != units:
             for element in self.elements:
                 element.rescale(units)
-            scl = getunitconversion(self.units, units)
-            self.matrix[0:3, 3] *= scl
```

Comment:
Why were these lines removed?
Just curious

If this is fixing a bug then we should add a unit test that fails before these lines were removed and that passes after they are removed.

Reply:
`self.matrix` is not defined anymore because we handle the transform on the Slicer side. so when calling `rescale` method it fails. This was not tested in `tests/test_transducer.py` but in my opinion it should not be part of this PR

Reply:
ah that's right, okay :+1: